tsconfig.json

```diff
@@ -0,0 +1,25 @@
+{
+  "compilerOptions": {
+    "allowJs": true,
+    "allowSyntheticDefaultImports": true,
+    "inlineSources": true,
+    "isolatedModules": true,
+    "jsx": "react",
+    "lib": ["dom", "es2020"],
+    "moduleResolution": "node",
+    "outDir": "tsout",
+    "rootDir": ".",
+    "sourceMap": true,
+    "strict": true
```

Comment:
Taking a quick peek of some other tsconfig.json file configs for chrome-extensions:
- https://github.com/chibat/chrome-extension-typescript-starter/blob/master/tsconfig.json
- https://betterprogramming.pub/creating-chrome-extensions-with-typescript-914873467b65

I think you might have explained this last Friday, but I forgot. Sorry if this was the case. Why are we not including
```
"module": "commonjs",
"target": "es6",
```
 again? 

----

"noEmitOnError" - might be something to consider later. This holds back from generating js files to `outDir` if there are ts errors found

Reply:
> I think you might have explained this last Friday, but I forgot. Sorry if this was the case. Why are we not including
>
> ```
> "module": "commonjs",
> "target": "es6",
> ```

We are, but I admit it's a bit hidden: that is being done by the `"extends": "@tsconfig/node14/tsconfig.json"` line. You can see the source for that [here](https://github.com/tsconfig/bases/blob/main/bases/node14.json).

> "noEmitOnError" - might be something to consider later. This holds back from generating js files to outDir if there are ts errors found

I don't mind adding it now! Added in 0ccb400a500e1d0683aa63a7c37c1f04b34978e5